.gitignore

```diff
@@ -8,4 +8,5 @@ ssl/serial*
 
 # terraform
 **/*.tfstate*
-**/.terraform
\ No newline at end of file
+**/.terraform
+.idea/
```

Comment:
`.idea/` shouldn't be here, we're trying to keep such stuff out of project's `.gitignore`.